examples/source/interactivity-dynamic-content/Subscription_Settings/index.html

```diff
@@ -0,0 +1,434 @@
+
+      Finally, we add a [placeholder and fallback](/documentation/guides-and-tutorials/develop/style_and_layout/placeholders) to keep the user informed about the state of the user interface and gracefully handle any issues.
+    -->
+    <amp-list src="<% base_path %>/subscription" binding="refresh" items="." single-item layout="fill">
+      <template type="amp-mustache">
+        <select>
+          {{#options}}
+            {{#isSelected}}
+              <option value="{{value}}" selected>
+                {{text}}
+              </option>
+            {{/isSelected}}
+            {{^isSelected}}
+              <option value="{{value}}">
+                {{text}}
+              </option>
+            {{/isSelected}}
+          {{/options}}
+        </select>
+      </template>
+      <div placeholder>Loading...</div>
+      <div fallback>Something went wrong. Please refresh</div>
+    </amp-list>
+
+    <!--
+      Note that the `amp-list` is annotated with [`single-item`](/documentation/components/amp-list#single-item) because our server responds to GET requests with a single logical value (the subscription setting) and we only want the template to be invoked once for the response. Setting [`items="."`](/documentation/components/amp-list#items) ensures that the template can access the response object's root fields.
+    -->
+
+    <!--
+      #### Step 3: Updating the Subscription Setting on Selection
+
+      To update the user's subscription setting on the server when they select a new setting, we start by wrapping the `select` element in an [AMP `form`](/documentation/components/amp-form) element, setting its `method` to `post`, and setting its `action-xhr` attribute to the URL where our server is listening for POST requests.
+
+      The `name` attribute of the `select` element is set to `nextSubscription` because `nextSubscription` is the name of the form data field where the server expects the user's new subscription setting to be in the POST request (see the [Server section](<% base_path %>/#server)) and the `select` element is the element containing the user's selected subscription setting.
+
+      Finally, we trigger the form submission when the `select` element's value is changed by using the [`on` attribute](/documentation/guides-and-tutorials/learn/common_attributes#on) to attach an event handler to the `select` element's [`change` event](/documentation/guides-and-tutorials/learn/amp-actions-and-events#input-elements) that invokes the `form` element's [`submit` action](/documentation/guides-and-tutorials/learn/amp-actions-and-events#form_1). Note that we gave the `form` element an `id` so that we could refer to it in the event handler.
```

Comment:
```suggestion
      Finally, we trigger the form submission when the `select` element's value changes. We do so by using the [`on` attribute](/documentation/guides-and-tutorials/learn/common_attributes#on) to attach an event handler to the `select` element's [`change` event](/documentation/guides-and-tutorials/learn/amp-actions-and-events#input-elements) , invoking the `form` element's [`submit` action]. [`submit` action](/documentation/guides-and-tutorials/learn/amp-actions-and-events#form_1). Note that we gave the `form` element an `id` so that we could refer to it in the event handler.
```